Sources/System/Internals/WindowsSyscallAdapters.swift

```diff
@@ -242,7 +258,12 @@ internal func mkdir(
     bInheritHandle: false
   )
 
-  if !CreateDirectoryW(path, &saAttrs) {
+  do {
+    if try !path.withCanonicalPathRepresentation({ path in CreateDirectoryW(path, &saAttrs) }) {
```

Comment:
It might be nicer as `guard try? .... else { }` I think.

Reply:
Done